crates/red_knot_python_semantic/src/types.rs

```diff
@@ -2352,18 +2445,42 @@ pub enum Truthiness {
 }
 
 impl Truthiness {
+    pub(crate) const fn from_bool(value: bool) -> Self {
+        if value {
+            Self::AlwaysTrue
+        } else {
+            Self::AlwaysFalse
+        }
+    }
+
     const fn is_ambiguous(self) -> bool {
```

Comment:
might be nice to make this `pub(crate)` as well, just for consistency with the other two you've added

```suggestion
    pub(crate) const fn is_ambiguous(self) -> bool {
```